src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationAboutToSubmitHandlerTest.java

```diff
@@ -62,14 +68,14 @@ public void setUp() {
     @Test
     public void givenCase_whenEventIsAmendApplication_thenCanHandle() {
         assertThat(handler
-                .canHandle(CallbackType.ABOUT_TO_SUBMIT, CaseType.CONSENTED, EventType.AMEND_APP_DETAILS),
+                .canHandle(CallbackType.ABOUT_TO_SUBMIT, CONSENTED, EventType.AMEND_APP_DETAILS),
             is(true));
     }
 
     @Test
     public void given_case_when_wrong_callback_then_case_can_not_handle() {
         assertThat(handler
-                .canHandle(CallbackType.ABOUT_TO_START, CaseType.CONSENTED, EventType.AMEND_APP_DETAILS),
+                .canHandle(CallbackType.ABOUT_TO_START, CONSENTED, EventType.AMEND_APP_DETAILS),
```

Comment:
same as above

Reply:
you can delete this method since the test has been included by`givenCase_whenEventIsAmendApplication_thenCanHandle`

Reply:
[cb00a1a](https://github.com/hmcts/finrem-case-orchestration-service/pull/1753/commits/cb00a1afefa3c10beac90cecd3181e7230463add)